PyNvCodec/TC/src/Tasks.cpp

```diff
@@ -513,37 +513,26 @@ int NvdecDecodeFrame::GetCapability(NV_DEC_CAPS cap) const
 
 namespace VPF
 {
-auto const format_name = [](Pixel_Format format) {
-  stringstream ss;
+#define TC_PIXEL_FORMAT_STRINGIFY(s) TC_PIXEL_FORMAT_STRINGIFY_(s)
+#define TC_PIXEL_FORMAT_STRINGIFY_(s) #s
```

Comment:
used stringview for this in #397 but this requires C++17. So plain C strings here